tools/pdf2txt.py

```diff
@@ -14,7 +15,190 @@
 from pdfminer.pdfparser import PDFParser
 
 
-def main():
+# Creating a new method for refactoring
+# (divide the main functions into different methods)
+def create_file(debug: int,
+                caching: bool,
+                outfile: str,
+                laparams: LAParams,
+                imagewriter: ImageWriter,
+                stripcontrol: bool,
+                scale: int,
+                layoutmode: str,
+                pagenos: Set[int],
+                maxpages: int,
+                password: str,
+                rotation: int,
+                encoding: str,
+                input_file: List[str],
```

Comment:
rename input_file to input_files since it is a List of files. Also since this parameter won't have a default value, set it as the first arugment

Reply:
Changed the name to input_files.